aws/auto-scaling-group/vpc.tf

```
@@ -106,23 +133,62 @@ resource "aws_route_table" "rt" {
   }
 
   tags = merge(var.tags, {
-    Name = "${var.name}-rt"
+    Name = "${var.name}-rt_public_subnets"
```

similar to above, here I think it is a good idea to add public or private
```suggestion
    Name = "${var.name}-public"
```

---

Yes, I agree, that's a good idea :)